core/src/banking_stage.rs

```diff
@@ -1096,15 +1120,9 @@ impl BankingStage {
         msgs: &Packets,
```

Comment:
Update comment on this method? Probably should remove:
```
// Also returned is packet indexes for transaction should be retried due to cost limits.
```

Reply:
updated the comment